### PR TITLE
linear_search_recursion.c

### PR DESCRIPTION
**Pull request by ankit-1912**
Linear Search Using Recursion

I am creating a pull request for...

- [ ] New algorithm

### CODE REVIEW

File: algorithms/searching/linear_search_recursion.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// C Program to implement recursive Linear Search 
+
+#include<stdio.h>
+#include<stdlib.h>
+/* Recursive function to search an element in arr[l..r] */
+int linearSearch(int arr[], int l, int r, int x)
```
Comment (hemanth-kotagiri) [inline algorithms/searching/linear_search_recursion.c:6]: This entire function along with the comments looks very similar to the one in the link mentioned.

**Review by hemanth-kotagiri — COMMENTED** [review]
Hello @ankit-1912. Thanks for your PR. But the code that you've written looks strikingly similar to the code in this [link](https://www.geeksforgeeks.org/recursive-c-program-linearly-search-element-given-array/) which is authored by someone else.

The snippets that I found plagiarized are commented individually by me and you can check them out.

Any PR found to be suspicious of plagiarised work will not be merged into the master of the upstream. Please try to implement the algorithms that interest you in your own unique way and raise a new PR.

A few points to remember before raising a PR:
1. Always remember to read contribution guidelines.
2. You can refer to any website for working on your PR, but directly copy-pasting code is not encouraged.
3. Always write clean and well-structured code with proper indentations.

Looking forward to your next PR.

Thanks!

**Comment by hemanth-kotagiri** [comment]
@abranhe, I believe, code should never be plagiarized from someone else.
I have reviewed this PR and have left the link of the same as well of which this PR seems to be suspicious of plagiarism

please create a new label: `plagiarism` and add it to this PR and close it if you agree with me on the same.

**Comment by ankit-1912** [comment]
Hey  @hemanth-kotagiri, Thanks for your email. Yes, I didn't read the
contribution guidelines and I admit my fault. Next time i won't commit the
same mistake. I'm closing my PR.



                                               Thank You!

**Comment by hemanth-kotagiri** [comment]
@ankit-1912 Good to hear that you admit your mistake. All the best!

Thanks!

**Pull request closed without merging**